security/seccomp-profiles.adoc

```diff
@@ -0,0 +1,77 @@
+[id="seccomp-profiles"]
+= Configuring seccomp profiles
+include::modules/common-attributes.adoc[]
+:context: seccomp-profiles
+
+toc::[]
+
+A container or an OpenShift pod runs a single application that performs one or more well-defined tasks. The application usually requires only a small subset of the underlying operating system kernel APIs.
+Seccomp (secure computing mode) Linux kernel feature can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating profile that is named as seccomp profile.
+
+?? Are below statements important or just a note?
+[IMPORTANT]
+====
+CRI-O runs unconfined by default, without any seccomp profile applied.
+====
+
+[IMPORTANT]
+====
+Seccomp profile cannot be applied to privileged containers.
+====
+
+== Configuring default seccomp profile
+Openshift ships with a default seccomp profile. On RHEL CoreOS, the default seccomp profile is stored in `/etc/crio/seccomp.json`. You can enable the default seccomp profile by overriding the empty seccomp value with `runtime/default`.
```

Comment:
```suggestion
OpenShift ships with a default seccomp profile referenced as `runtime/default`. On RHEL CoreOS, the default seccomp profile is stored in `/etc/crio/seccomp.json`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` in the `securityContext.seccomp.type` field or by using the deprecated Pod annotations `seccomp.security.alpha.kubernetes.io/pod: runtime/default` and `container.seccomp.security.alpha.kubernetes.io/[name]: runtime/default`.
```

Reply:
@haircommander do we still use `/etc/crio/seccomp.json` in recent OpenShift versions or do we use the in-memory fallback of containers/common?

Reply:
@haircommander - Can you confirm the above query?

@saschagrunert - Do these deprecated annotations apply to 4.6+ considering I am documenting only for 4.6+ documents.

Reply:
OpenShift 4.8 indicates that it's using the in-memory profile rather than `/etc/crio/seccomp.json`:
```
Sep 02 12:09:53 ip-10-0-147-62 crio[642212]: time="2021-09-02 12:09:53.418478872Z" level=info msg="No seccomp profile specified, using the internal default"
```

Reply:
I have made these changes. Can you mention what is [name] here `container.seccomp.security.alpha.kubernetes.io/[name]: runtime/default `? Name of?